### DISCUSSION TITLE
ETA version 5 available?

### DISCUSSION THREAD

**Discussion by kramer65** [cantaloupe-project/cantaloupe#460] [General]
We're waiting for some of the features that are coming in version 5; specifically `getPageCount()` to get the number of pages from a PDF.

Is there any estimation known for when v5 will be released? Or when something like release candidates will start appearing?

Are we talking weeks, months or years?

**Comment by adolski** [comment]
Hi @kramer65, I can't commit to a firm date, but probably in the next couple months.

I can say that what's in place now on `develop` is very close to what 5.0 will be, and I know that some people are already using it. There may be a few new config keys coming (e.g. #434) but not much else, I would imagine.

**Comment by kramer65** [comment]
@adolski - Thanks for the information. In that case we'll discuss using develop in our infrastructure.

Just one more thing; does Cantaloupe use release candidates, or will develop at one point simply be designated 5.0?
